integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -1433,122 +1429,6 @@ fn test_archival_save_trie_changes() {
     }
 }
 
-fn test_archival_gc_common(
```

Comment:
Why delete those tests? Can you run the gc from the main test thread manually to make it work?

Reply:
I can, but I am not sure what that would achieve. Essentially it requires implementing a separate logic that manually invokes `clear_data` for archival nodes. However, if we want to test the underlying `clear_data` implementation, there are already integration tests that only constructs `Chain` which achieves that. If we want to test that archival node as a whole function as expected (doesn't delete data), then `test_garbage_collection_archival.py` achieves that. A test here feels like it is not testing any interfaces. If there is a bug in how `GCActor` runs, then it may very well happen that a test here passes but archival nodes do not work properly. I think what would help is to put `test_garbage_collection_archival.py` into CI. @andrei-near what is the best way to do that?

Reply:
Fixed